lib/rubocop/cop/style/documentation.rb

```diff
@@ -3,10 +3,11 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks for missing top-level documentation of
-      # classes and modules. Classes with no body are exempt from the
-      # check and so are namespace modules - modules that have nothing in
-      # their bodies except classes, other modules, or constant definitions.
+      # This cop checks for missing top-level documentation of classes and
```

Comment:
I wonder if we shouldn't add here examples of the code that's exempt, so it's clearer for users what the cop acts on and what it ignores.

Reply:
Ok, I think you're right. I will do it soon.

Reply:
@bbatsov Sorry for the time spent. What do you think of comments? That's enough for you as an explanation?